src/main/java/dog/pawbook/model/dog/Gender.java

```diff
@@ -0,0 +1,37 @@
+package dog.pawbook.model.dog;
+
+import static java.util.Objects.requireNonNull;
+
+public class Gender {
```

Comment:
Maybe we can avoid the dilemma just by calling it `Sex`, since gender is what you identify with, sex is biological, easier to just have an other instead.

Reply:
I agree, will amend this!